episodes/superspreading-simulate.Rmd

```diff
@@ -0,0 +1,851 @@
+
++ Greater variation reduces the overall probably of a single new case causing a large local outbreak, because most cases infect few others and individuals that generate a large number of secondary cases are relatively rare.
+
++ However, if a 'superspreading event' does occur and the outbreak gets established, this variation can make an outbreak harder to control using *mass interventions* (i.e. blanket interventions that implicitly assume everyone contributes equally to transmission), because some cases contribute disproportionality: a single uncontrolled case may generate a large number of secondary cases.
+
++ Conversely, variation in transmission may provide opportunities for *targeted interventions* if the individuals who contribute more to transmission (due to biological or behavioural factors), or the settings in which 'superspreading events' occur, share socio-demographic, environmental or geographical characteristics that can be defined.
+
+<!-- what we dont know -->
+
+How can we quantify the potential of a new infection to cause a large outbreak based on its reproduction number $R$ and the dispersion $k$ of its offspring distribution?
+
+```{r,echo=FALSE,warning=FALSE,message=FALSE,fig.cap="Observed number of cumulative cases from the Middle East respiratory syndrome (MERS) outbreak in South Korea, 2015, alongside with simulated transmission chains assuming an offspring distribution with $R=0.6$ and $k=0.02$."}
+library(epichains)
+library(epiparameter)
+library(tidyverse)
+library(outbreaks)
+
+mers_cumcases <- mers_korea_2015$linelist %>%
+  # incidence2 workflow
+  incidence2::incidence(date_index = "dt_onset") %>%
+  incidence2::complete_dates() %>%
+  # wrangling using {dplyr}
+  mutate(count_cumsum = cumsum(count)) %>%
+  rownames_to_column(var = "day") %>%
+  mutate(day = as.numeric(day))
+
+# offspring distribution
+mers_offspring <- c(mean = 0.60, dispersion = 0.02)
+
+# generation time
+serial_inverval <- epidist_db(
+  disease = "mers",
+  epi_dist = "serial",
+  single_epidist = TRUE
+)
+
+# Set seed for random number generator
+set.seed(33)
+# Number of simulation runs
+number_chains <- 1000
+# Number of initial cases
+initial_cases <- 1
+
+simulated_chains_map <-
+  # iterate one function across multiple numbers (chain IDs)
+  map(
+    # vector of numbers (chain IDs)
+    .x = seq_len(number_chains),
+    # function to iterate to each chain ID number
+    .f = function(sim) {
+      simulate_chains(
+        # simulation controls
+        index_cases = initial_cases,
+        statistic = "size",
+        # offspring
+        offspring_dist = rnbinom,
+        mu = mers_offspring["mean"],
+        size = mers_offspring["dispersion"],
+        # generation
+        generation_time = function(x) generate(x = serial_inverval, times = x)
```

Comment:
```suggestion
        generation_time = function(x) generate(x = serial_interval, times = x)
```

Reply:
fixed in https://github.com/epiverse-trace/tutorials-middle/commit/171028626419b00712a2f70699b319900431875c